build/room_ensurer/room_ensurer.py

```diff
@@ -237,14 +237,29 @@ def _get_room(self, server_name: str, room_alias_prefix: str) -> Optional[RoomIn
 
     def _create_server_user_power_levels(self) -> Dict[str, Any]:
 
-        server_admin_power_levels: Dict[str, Dict[str, int]] = {"users": {}}
+        server_admin_power_levels: Dict[str, (Dict[str, int], int)] = {
```

Comment:
The type here is not correct, not all values are dicts, e.g. ` "users_default": MatrixPowerLevels.USER,`.

Assuming that these constants are `int`s, the correct type would be `Dict[str, Union[int, Dict[str, int]]]`